docs/release-notes/Percona-Everest-0.10.1-(2024-05-16).md

```diff
@@ -0,0 +1,47 @@
+# What's new in Percona Everest 0.10.1
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source cloud native database platform that helps provision and manage databases faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 0.10.1 introduces the following changes:
+
+
+## Fixed issues
+
+### Retention copies
+
+- [EVEREST-1054](https://perconadev.atlassian.net/browse/EVEREST-1054) - While setting up a PostgreSQL database cluster, we encountered an issue with the retention copies not working when using the minIO bucket for backups and restores. This problem resulted in the restored cluster becoming unresponsive and entering an unknown state. The issue has been resolved now.
```

Comment:
Actually this issue has nothing to do with retention copies. The issue was that in PG we weren't able to restore from backups stored in a storage with a self-signed certificate or that only supports path-style request URL.
So it's basically the same issue as EVEREST-1082 that we mention in the "Restores" section. I think we should move this issue to that section and we have a similar description to the one for EVEREST-1082